Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -178,5 +187,17 @@ - (void)reset
     SEGLog(@"[Amplitude regnerateDeviceId];");
 }
 
+#pragma utils
+
+- (void)incrementTrait:(NSString *)trait andValue:(NSString *)value
+{
+    NSArray *increments = [self.settings objectForKey:@"traitsToIncrement"];
+    for (NSString *increment in increments) {
```

Comment:
This seems inefficient.  The runtime is O(n*m) where n = size of `payload.traits` and m = size of `settings.traitsToIncrement`.

You should remove the case insensitivity and we can cut the runtime to O(n).